sdcm/utils/csrangehistogram.py

```diff
@@ -229,25 +229,62 @@ def build_from_log_line(self, log_line: str, hst_log_start_time: float) -> dict[
 
         return _CSRangeHistogramBuilder._get_summary_for_operation_by_hdr_tag(histogram)
 
-    def _build_histogram_from_file(self, hdr_file: str, hdr_tag: str) -> _CSRangeHistogram:
+    @staticmethod
+    def switch_tag(hdr_tag):
+        tag_pairs = [[_CSHistogramThroughputTags.WRITE.value, CSHistogramTags.WRITE.value],
+                     [_CSHistogramThroughputTags.READ.value, CSHistogramTags.READ.value]]
+        for pair in tag_pairs:
+            if hdr_tag in pair:
+                return [tag for tag in pair if tag != hdr_tag][0]
+        return None
+
+    def _build_histogram_from_file(self, hdr_file: str, hdr_tag: str) -> _CSRangeHistogram | None:
+        def analyze_hdr_file():
```

Comment:
would  be nice to have return value documented for this internal one

Reply:
added